src/markdown-pages/docs/ecosystem/ecosystem.md

```diff
@@ -0,0 +1,32 @@
+---
+path: "/docs/ecosystem"
+title: "The Galasa Ecosystem"
+---
+
+To realise the full power of Galasa, you need to run your tests inside a Galasa ecosystem.
+
+The SimBank tests showcase how the Galasa framework can run inside a locally hosted JVM. Local runs use their own Galasa property files to act as configurational services, making it easy to instantiate and run tests. 
+
+However, there are limitations to taking a local-only approach:
+
+- Configuration settings and test results cannot be shared across teams
```

Comment:
configuration settings and test results, artefacts are stored locally, meaning they cannot be shared